src/main/java/com/intuit/graphql/orchestrator/federation/EntityFetchingException.java

```diff
@@ -29,6 +31,8 @@ public static class Builder extends BuilderBase<EntityFetchingException.Builder,
     private String parentTypeName;
     private String serviceNameSpace;
 
+    private String additionalInfo;
```

Comment:
I'm thinking of re writing this class such that
1. message is passed by the use of the class
2. metadata is part of the extensions instead of the message.  
Thoughts?